res/database/Default/user/cujo.xml

```diff
@@ -461,7 +462,7 @@
 				<en>Markus Trust, who embarrassed the Russian Air Force a few days ago with his Schessnah landing on Moscow's Red Square, was sentenced by a judge to 5 years in a labor camp for serious hooliganism.</en>
 			</description>
 			<effects>
-				<effect trigger="happen" type="triggernews" time="2,5,6,18,22" news="cujo-news-roter-platz-07" />
+				<effect trigger="happen" type="triggernews" time="2,1,1,10,16" news="news-jorgaeff-1987-0-UdSSR-01b" />

```

Comment:
trigger jorgaeff-news in addition to "cujo-news-roter-platz-07" (simply add second effect trigger)

Reply:
I made it linear, the jorgaff-news thread then jumps back to the cujo one.

Reply:
My thought was rather that the two run in parallel, because what happens with Markus and the ministers need not be linear.

Reply:
ok made it parallel